src/radar-chart/index.js

```diff
@@ -53,7 +54,7 @@ function getAxes(props) {
     hideInnerMostValues
   } = props;
   return domains.map((domain, index) => {
-    const angle = (index / domains.length) * Math.PI * 2 + startingAngle;
```

Comment:
this is just a nit, but i tend to think the parens make this type of calculation more legible, would you mind putting them back?

Reply:
I completely agree. I accidentally had a very opinionated `JSPrettier` running that didn't like the extra parens. All 3 instances have been fixed 👍